src/UI/Pages/Home/EnvironmentsOverview.test.tsx

```diff
@@ -106,3 +106,10 @@ test("Given environments overview When filtering by name and project Then only t
     })
   ).toBeVisible();
 });
+
+test("Given environments overview When rendering environment with icon Then the icon is shown", async () => {
+  const { component } = setup();
+  render(component);
+  const cardsWithIcons = await screen.findByRole("img");
```

Comment:
This seems like a very broad query. Not sure if adding a test for the alt value is useful here.
Maybe test both the presence and absence of the image for 2 separate environments?